carmin.yaml

```diff
@@ -456,6 +582,7 @@ definitions:
   Execution:
     type: object
     required:
+      - identifier
```

Comment:
I remember having a headache about that. It's complicated because it should not be present in the execution creation body. I've decided to mark it a readonly. I've not checked if swagger has improved that, but there's no ideal solution in my knowledge.

Reply:
ok, I'll remove it.